src/Umbraco.Web.UI.Client/src/views/content/move.html

```diff
@@ -76,9 +76,9 @@
 
 
     <div class="umb-dialog-footer btn-toolbar umb-btn-toolbar" ng-hide="success">
-        <a class="btn btn-link" ng-click="close()" ng-show="!busy">
+        <button type="" class="btn btn-link" ng-click="close()" ng-show="!busy">
```

Comment:
Any reason this is not `<button type="button"` @BatJan ?

Reply:
Looks like an oversight because I had a an editor plugin that kinda worked and helped autocomplete stuff...except from when it did not 🤦‍♂ 😄 - I will add the types when I find a litle amount of time 👍

Reply:
@BatJan No problem, it was just this line that was missing it, I was just curious if it was on purpose. I can add it as well when I check out the PR!